src/infinity-menu.js

```diff
@@ -261,6 +261,7 @@ export default class InfinityMenu extends React.Component {
 
 		/*header component*/
 		const headerProps = {
+			metadata: this.props.metadata,
```

Comment:
you might also want to add tests to make sure this is having the desired effect.